DATA/production/workflow-multiplicities.sh

```diff
@@ -100,7 +100,7 @@ if [[ ! -z ${OPTIMIZED_PARALLEL_ASYNC:-} ]]; then
   elif [[ $OPTIMIZED_PARALLEL_ASYNC == "PbPb_4gpu" ]]; then
     [[ -z $TIMEFRAME_RATE_LIMIT ]] && TIMEFRAME_RATE_LIMIT=20
     [[ -z $SHMSIZE ]] && SHMSIZE=128000000000 # SHM_LIMIT 3/4
-    [[ -z $TIMEFRAME_SHM_LIMIT ]] && TIMEFRAME_SHM_LIMIT=$(($SHMSIZE / 3))
+    [[ -z ${TIMEFRAME_SHM_LIMIT+x} ]] && TIMEFRAME_SHM_LIMIT=$(($SHMSIZE / 3))
```

Comment:
Hello @davidrohr , 
Please check. Without this, in async we get:
```
/home/zampolli/SOFT/alibuild/ali-o2-dev1/sw/ubuntu2004_x86-64/O2DPG/master-local2/DATA/production/workflow-multiplicities.sh: line 105: TIMEFRAME_SHM_LIMIT: unbound variable
```

Reply:
Hello @davidrohr , 
Is this ok for you?
Chiara

Reply:
I would prefer `${TIMEFRAME_SHM_LIMIT:-}` to be in line with the other checks for optional variables.
And actually, I think we should do the same for all the checks for `SHMSIZE` and `TIMEFRAME_RATE_LIMIT` in this file. Currently it works in the async reco only since you set it explicitly, but setting the defaults this way would not work.
Can you do this in your PR, or shall I do it?

Reply:
Actually, that is how I implemented the first time, but then it was not working. I believe that I was using the wrong branch by mistake, so let me retry. 
I can then add the others too.